src/gpl/CMakeLists.txt

```diff
@@ -85,6 +85,32 @@ target_link_libraries(gpl
     rsz
     grt
 )
+# If CUDA is not installed there will not be cmake
+# support for the package so this needs to be "quiet".
+find_package(CUDA QUIET COMPONENTS Core Widgets)
```

Comment:
This is marked deprecated (see https://cmake.org/cmake/help/latest/module/FindCUDA.html). 

See also https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html

Reply:
Hi Matt, this is a bit weird. I deleted find_package() then there appeared many many CUDA errors. Once I added "find_pakage()" back, it compiles successfully again. Some websites said some third-parties still depend on "find_package()". I will research more on this problem.

Reply:
This is not resolved

Reply:
find_package() is the simplest command to check if any of CUDA_capable GPUs exists. 
I have deleted this line, then here we just check if GPU is enabled. If GPU is enabled but CUDA is not found, initialplace.cpp will turn the program to CPU solver and give a warning.

Reply:
find_package() is the simplest command to check if any of CUDA_capable GPUs exists. The true value of (CUDA_found) allows the .cu file and CUDA libraries to be linked. I think we should keep this sentence.
![image](https://user-images.githubusercontent.com/57721574/183704356-e2008ed0-82f2-44a2-b836-4f2eaf790c90.png)

Reply:
FindCUDAToolkit requires cmake 3.17.  We currently only require 3.9 but could easily update.  However Ubuntu 20.04 LTS ships with 3.16.3 which means updating to 3.17 would require a special install.  In light of that we can leave this for now.  Hopefully people will migrate to 22.04 LTS with 3.22.1 and this can be updated.

Reply:
@maliberty @RuijiaoSun Let's add a comment with the requirements to remove this, and what the replacement is.

i.e. OpenROAD only supports Ubuntu 22.04+ distros, and Cmake 3.17+

Reply:
![image](https://user-images.githubusercontent.com/57721574/183758565-65ecb181-056a-4189-b938-fcf0b78cade0.png)
Is this okay?

Reply:
That seems good.